cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/internal/SSAConstruction.qll

```diff
@@ -4,23 +4,35 @@ private import semmle.code.cpp.ir.implementation.Opcode
 private import semmle.code.cpp.ir.internal.OperandTag
 private import NewIR
 
+private class OldBlock = Reachability::ReachableBlock;
+private class OldInstruction = Reachability::ReachableInstruction;
```

Comment:
Do the benefits outweigh the cost of running the reachability and constant analyses three times? There will certainly be cases where it becomes better when iterated, but it's also hundreds of lines of extra code we'll be running every time. Using pyrameterized modules also comes with a maintainability cost.

Reply:
We actually only run it twice: Once on raw IR (used when building unaliased_ssa), and once on unaliased_ssa (used when building aliased_ssa). I've now removed the aliased_ssa instantiation, since it was unused.